engines/billing/questions.md

```diff
@@ -0,0 +1,22 @@
+Questions
+====================
+
+1. Should admin be able to see subscriptions?
+
+2. Should user be able to upgrade or downgrade subscriptions?
```

Comment:
Yep, Braintree know if there is one or more plans. But they cannot be created, updated, or deleted via the API, only through the Control Panel. Subscriptions depend on plan_id. To create a subscription, you only need a stored payment_method_token and a plan_id, which must be created via the Control Panel. The subscription will be created using the price, trial duration (if any), billing cycle, and any add-ons or discounts specified within the plan's details.